### PR TITLE
Master

### PR DESCRIPTION
**Pull request by prembala1710**

### CODE REVIEW

**Review by prem-bkp-test-github — APPROVED** [review]

File: .github/actions/CliqInformer/action.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: "CliqInformer"
-description: "The Github Action is used to Integrate Github and Zoho Cliq by Notifying about the Github Events Performed to the Zoho Cliq Channels. Mention (me) to Link your Github Account and (repo) to link your Github Repository and (workflow) to link the workflow to be Performed in the Message such as 'A Github Action has been Performed at (repo) by (me)' will send a message as 'A Github Action has been Performed at https://www.github.com/user_name/repository_name by user_name'"
```
Comment (prembala1710) [inline .github/actions/CliqInformer/action.yml:2]: Need Explanation Regarding This Change

Comment (prembala1710) [reply .github/actions/CliqInformer/action.yml:2]: Check Once

**Review by prembala1710 — COMMENTED** [review]
Check Clearly and Report Soon

File: .github/actions/CliqInformer/action.yml [hunk 2]
```diff
@@ -63,11 +63,11 @@ inputs:
   pull-request-review-comment-message:
     description: "The Message to be Delivered to the Cliq Channel during a pull request review comment event"
   pull-request-target-message:
-    description: "The Message to be Delivered to the Cliq Channel during a pull request comment event"
```
Comment (prembala1710) [inline .github/actions/CliqInformer/action.yml:66]: Need Explaination

Comment (prembala1710) [reply .github/actions/CliqInformer/action.yml:66]: Faster

**Review by prembala1710 — COMMENTED** [review]
Need Quick Response

**Pull request merged**